src/frontend/css/style.scss

```diff
@@ -542,9 +542,23 @@ to add animation */
 }
 
 /* page form styling */
-#slug-div {
-  input:read-only {
-      --tw-bg-opacity: 1;
-      background-color: rgba(229, 231, 235, var(--tw-bg-opacity));
+#content_form {
+  #link-container {
+    div {
+      --tw-text-opacity: 1;
+      color: rgba(59, 130, 246, var(--tw-text-opacity));
```

Comment:
I don't think it's a good idea to hardcode tailwind colors in the stylesheet. Why not directly toggling the class `text-blue-500`? I think the stylesheet should only be used if something is not achievable with tailwind.

Reply:
So the main reason for me to involve this file was, that I couldn't find a tailwind feature for pseudo-class read-only, but this pseudo-class is just required for the input element (background toggling), but for the text color we can take tailwind as well